pkg/values/packagemaps.go

```diff
@@ -58,6 +58,148 @@ type ModelPackageMap map[DistroFamilyInterface]map[Architecture]map[Model]Versio
 // VersionMap is a map of a constraint to a list of packages
 type VersionMap map[string][]string
 
+// BasePackagesModels allows model-specific base packages per distro/arch/model
+var BasePackagesModels = ModelPackageMap{
+	Ubuntu: {
+		ArchARM64: {
+			AgxOrin: {
+				Common: {
+					"bridge-utils",
+					"efibootmgr",
+					"fuse",
+					"git",
+					"grub2-common",
+					"libssl-dev",
+					"policykit-1",
+					"vim",
+				},
+			},
+			OrinNX: {
+				Common: {
+					"bc",
+					"binutils",
+					"bluez",
+					"bridge-utils",
+					"bzip2",
+					"ca-certificates",
+					"can-utils",
+					"cloud-guest-utils",
+					"conntrack",
+					"console-data",
+					"console-setup",
+					"coreutils",
+					"cron",
+					"cryptsetup",
+					"curl",
+					"debianutils",
+					"default-jdk",
+					"dirmngr",
+					"dmsetup",
+					"dosfstools",
+					"dracut",
+					"dracut-live",
+					"dracut-network",
+					"e2fsprogs",
+					"efibootmgr",
+					"ethtool",
+					"fail2ban",
+					"fdisk",
+					"file",
+					"fuse3",
+					"gawk",
+					"gdisk",
+					"gettext",
+					"gnupg",
+					"gnupg1-l10n",
+					"gpg-agent",
+					"grub2-common",
+					"grub-efi-arm64",
+					"grub-efi-arm64-bin",
+					"grub-efi-arm64-signed",
+					"haveged",
+					"iproute2",
+					"iptables",
+					"iputils-ping",
+					"isc-dhcp-client",
+					"isc-dhcp-common",
+					"isc-dhcp-server",
+					"jq",
+					"kbd",
+					"kmod",
+					"lbzip2",
+					"less",
+					"libatm1",
+					"libglib2.0-data",
+					"libgpm2",
+					"libldap-common",
+					"libnss-systemd",
+					"libopencv-dev",
+					"libpam-cap",
+					"libqt5core5a",
+					"libsasl2-modules",
+					"libssl-dev",
+					"lldpd",
+					"logrotate",
+					"lvm2",
+					"mdadm",
+					"modemmanager",
+					"mtd-utils",
+					"nano",
+					"nbd-client",
+					"ncurses-term",
+					"netplan.io",
+					"net-tools",
+					"networkd-dispatcher",
+					"network-manager",
+					"nfs-client",
+					"nfs-common",
+					"nftables",
+					"open-iscsi",
+					"openssh-server",
+					"open-vm-tools",
+					"os-prober",
+					"packagekit-tools",
+					"parted",
+					"passwd",
+					"patch",
+					"pigz",
+					"pkg-config",
+					"policykit-1",
+					"psmisc",
+					"publicsuffix",
+					"qemu-user-static",
+					"rsync",
+					"shared-mime-info",
+					"shim-signed",
+					"snmpd",
+					"software-properties-common",
+					"squashfs-tools",
+					"ssh",
+					"sudo",
+					"systemd",
+					"systemd-container",
+					"systemd-sysv",
+					"systemd-timesyncd",
+					"tar",
+					"tpm2-tools",
+					"ubuntu-advantage-tools",
+					"udev",
+					"unzip",
+					"vim",
+					"wget",
+					"wireless-tools",
+					"wpasupplicant",
+					"xxd",
+					"xz-utils",
+					"zerofree",
+					"zfsutils-linux",
+					"zstd",
```

Comment:
this are all the packages in https://github.com/kairos-io/kairos/blob/master/images/Dockerfile.nvidia-orin-nx I assume it can be stripped down to only the necessary ones, but I'm not sure what would those be 😅 ... I just know we probably don't need nano and vim there hehe 

Opinions @jordankrp @mudler ?